sources/platform/actors/development/actor_definition/dockerfile.md

```diff
@@ -1,11 +1,10 @@
 ---
-title: Base Docker images
+title: Dockerfile
 description: Learn about the Docker images you can use as your actor's base. Choose the correct base image for your actor's type and requirements.
-slug: /actors/development/base-docker-images
+slug: /actors/development/actor-definition/dockerfile
+sidebar_position: 4
 ---
 
-# Base Docker images {#base-docker-images}
```

Comment:
I can't comment on unchanged lines, so adding it here.

- The latest images are on Node 18 I believe.
- This no longer applies: `Note that some images available in the Apify UI can be marked as deprecated. This means that they should no longer be used for new projects and old projects are encouraged to migrate to one of the non-deprecated images.`

Reply:
https://github.com/apify/apify-docs/issues/568

> Note that some images available in the Apify UI can be marked as deprecated. This means that they should no longer be used for new projects and old projects are encouraged to migrate to one of the non-deprecated images.

Removed.